### ISSUE TITLE
Weight of the model

### ISSUE THREAD

**Issue by while-lin** [NasimShafiee/Introvert-Human-trajectory-prediction-via-conditional-3d-attention#2]
Hi,
thank you very much for your nice work and sharing it!

Could You share the weight of the model on ETH/UCY? Thinks!

**Comment by while-lin** [comment]
AND I have another issues
In the dataset you give,I guess a pedestrian's trajectory may be used for training several times.
![image](https://user-images.githubusercontent.com/52849152/198845616-8fa77dcd-e99f-4587-8ffd-acd4a9fd984c.png)
IN the pic above, Ped have 22 timesteps in the video, and the 22 timesteps Produced three slice, which is (0~20),(1~21),and (2~22), IS that ok?

**Comment by NasimShafiee** [comment]
Each datapoint contains 20 timesteps, 8 observations and 12 predictions.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by while-lin** [comment]
yes i know that. Well, I mean, the pos_xy of (1~20)timesteps in data_id0 is euqal to the pos_xy of (1~20)timesteps in data_id1, and I don't know whether it is reasonable to split the row data in this way.

**Comment by while-lin** [comment]
yes i know that. Well, I mean, the pos_xy of (1-20)timesteps in data_id0 is euqal to the pos_xy of (0-19)timesteps in data_id1, and I don't know whether it is reasonable to split the row data in this way
